algorithms/src/snark/varuna/ahp/prover/round_functions/mod.rs

```diff
@@ -64,6 +64,13 @@ impl<F: PrimeField, SM: SNARKMode> AHPForR1CS<F, SM> {
             randomizing_assignments.push(circuit_assignments);
         }
 
+        if randomizing_assignments.len() != circuits_to_constraints.len() {
+            return Err(AHPError::AnyhowError(anyhow::anyhow!(
+                "[prover Init] Expected {} randomizing assignments, but {} were provided.",
```

Comment:
Nit. Should the print format be consistent with the others in this file?

Ditto for all other error messages and prints added to the `varuna/` directory.